tests/gem5/se_mode/rvv_intrinsic_tests/test.py

```diff
@@ -0,0 +1,45 @@
+import os
+import re
+import sys
+
+from testlib import *
+
+binaries = [
+    "rvv_branch",
+    "rvv_index",
+    "rvv_matmul",
+    "rvv_memcpy",
+    "rvv_reduce",
+    "rvv_saxpy",
+    "rvv_sgemm",
+    "rvv_strcmp",
+    "rvv_strcpy",
+    "rvv_strlen",
+    "rvv_strlen-fault",
+    "rvv_strncpy",
+]
+
+vlens = [2**x for x in range(7, 15)]
+
+if not (src_dir := os.environ.get("SRCDIR")):
+    print("SRCDIR env var not defined. Exiting.", file=sys.stderr)
+    exit()
+
+for binary in binaries:
+    bin_path = f"{src_dir}/apps/rvv-intrinsic-tests/bin/{binary}"
+
+    if not os.path.isfile(bin_path):
+        print(f"Binary {bin_path} does not exist. Skipping.", file=sys.stderr)
+        continue
+
+    out_verifier = verifier.MatchRegex(re.compile(f"^{bin_path}: pass$"))
+    for vlen in vlens:
+        gem5_verify_config(
+            name=f"test-riscv-{binary}-vlen_{vlen}-O3-se-mode",
+            fixtures=(),
+            verifiers=(out_verifier,),
+            config=f"{config.base_dir}/configs/example/riscv-se.py",
```

Comment:
Please put the config file in the tests repository. I would prefer to not have tests depend on scripts outside of the tests directory (unless the test is *testing the script*).